src/game/DIFFICULTIES.ts

```diff
@@ -11,7 +12,7 @@ const DIFFICULTIES = {
 };
 export default DIFFICULTIES;
 
-export function getLabel(difficulty) {
+export function getLabel(difficulty: number | undefined) {
```

Comment:
It's prefered to use this syntax `difficulty?: number` this way users of the function don't have to pass undefined but can just ignore the param.